lib/jxl/decode_test.cc

```diff
@@ -1710,6 +1711,42 @@ TEST(DecodeTest, PixelTestOpaqueSrgbLossyNoise) {
   }
 }
 
+TEST(DecodeTest, ProcessEmptyInputWithBoxes) {
+  size_t xsize = 123, ysize = 77;
+  std::vector<uint8_t> pixels = jxl::test::GetSomeTestImage(xsize, ysize, 3, 0);
+  jxl::CompressParams cparams;
+  uint32_t channels = 3;
+  JxlPixelFormat format = {channels, JXL_TYPE_FLOAT, JXL_LITTLE_ENDIAN, 0};
+  for (int i = 0; i < kCSBF_NUM_ENTRIES; ++i) {
+    JxlDecoder* dec = JxlDecoderCreate(NULL);
+    CodeStreamBoxFormat box_format = (CodeStreamBoxFormat)i;
+    printf("Testing empty input with box format %d\n", (int)box_format);
+    jxl::PaddedBytes compressed = jxl::CreateTestJXLCodestream(
+        jxl::Span<const uint8_t>(pixels.data(), pixels.size()), xsize, ysize, 3,
+        cparams, box_format, JXL_ORIENT_IDENTITY, /*add_preview=*/false,
+        /*add_intrinsic_size=*/false);
+    const int events =
+        JXL_DEC_BASIC_INFO | JXL_DEC_FULL_IMAGE | JXL_DEC_COLOR_ENCODING;
+    EXPECT_EQ(JXL_DEC_SUCCESS,
+              JXL_DEC_SUCCESS != JxlDecoderSubscribeEvents(dec, events));
```

Comment:
Why the != here?

Reply:
oops, this was just leftover somehow. Fixed.